lib/graphql.rb

```diff
@@ -94,8 +94,7 @@ class << self
   autoload :CoercionError, "graphql/coercion_error"
   autoload :InvalidNameError, "graphql/invalid_name_error"
   autoload :IntegerDecodingError, "graphql/integer_decoding_error"
-  autoload :IntegerEncodingError, "graphql/integer_encoding_error"
-  autoload :StringEncodingError, "graphql/string_encoding_error"
+  autoload :LegacyScalarCoercionError, "graphql/legacy_scalar_coercion_error"
   autoload :DateEncodingError, "graphql/date_encoding_error"
   autoload :DurationEncodingError, "graphql/duration_encoding_error"
```

Comment:
Side note, I believe both of these should be `Decoding` since they are only used in `coerce_input` methods. I can fix them separately.